workers/unity/Assets/Playground/Scripts/Worker/AndroidClientWorkerConnector.cs

```diff
@@ -17,9 +19,11 @@ public class AndroidClientWorkerConnector : MobileWorkerConnector, IMobileConnec
         [SerializeField] private GameObject level;
 
         private GameObject levelInstance;
+        private ConnectionService connectionService;
 
-        public async void TryConnect()
+        public async void TryConnect(ConnectionService connectionService)
```

Comment:
I think @samiwh noted on this in another PR - but should async methods have async in the name. I.e. - `TryConnectAsync`